projects/ad9361/src/parameters.h

```diff
@@ -122,9 +122,14 @@
 #define ADC_DDR_BASEADDR			XPAR_DDR3_SDRAM_S_AXI_BASEADDR + 0x800000
 #define DAC_DDR_BASEADDR			XPAR_DDR3_SDRAM_S_AXI_BASEADDR + 0xA000000
 #else
+#if defined(XPAR_AXI_DDR_CNTRL_BASEADDR)
```

Comment:
this looks odd, and not very deductive. If you need to differentiate between boards you should go for the approach:
https://github.com/analogdevicesinc/no-OS/blob/6d75bc376eb7f1a4fd992cac75a7e33c0c89e37e/projects/ad9361/builds.json#L60